script.js

```diff
@@ -1,40 +1,84 @@
-function colorClick(event) {
+function colorSelected({ target }) {
```

Comment:
Pensa que os nomes das funcoes precisam deixar mais claro para quem bater o olho o que ela faz. `colorSelected` ta no passado e traz mais a ideia de acessar uma propriedade do que selecionar ou fazer algo no ato de selecionar uma cor.

Pensei nessas sugestoes aqui, ai voce ve o que acha.

```suggestion
function onColorSelect({ target }) {
```

```suggestion
function handleColorSelect({ target }) {
```

Reply:
Faz sentido. Vou usar as suas sugestões